draft-irtf-cfrg-frost.md

```diff
@@ -554,20 +577,20 @@ for more information.
 Because key generation is not specified, all signers are assumed to have the (public) group state that we refer to as "group info"
 below, and their corresponding signing key shares.
 
-In particular, it is assumed that the Coordinator and each signing participant `P_i` knows the following
+In particular, it is assumed that the Coordinator and each signer participant `P_i` knows the following
 group info:
 
-- Group public key, an `Element` in `G`, denoted `PK = G.ScalarMultBase(s)`, corresponding
-  to the group secret key `s`, which is a `Scalar`. `PK` is an output from the group's
-  key generation protocol, such as `trusted_dealer_keygen`or a DKG.
-- Public keys for each signer, denoted `PK_i = G.ScalarMultBase()`, which are similarly
-  outputs from the group's key generation protocol, `Element`s in `G`.
+- Group public key, an `Element` in `G`, denoted `PK`, corresponding
+  to the group secret key `s`, which is a `Scalar`. `PK` is an output from the group's key generation protocol, such as `trusted_dealer_keygen` or a DKG.
+- Public keys for each signer, denoted `PK_i`, which are similarly outputs
+  from the group's key generation protocol, `Element`s in `G`.
```

Comment:
Should there be an s after `Element` here?

Reply:
It's meant to be plural (this isn't a new change).